src/PythonClient/src/quixstreams/models/timeseriesdata.py

```diff
@@ -178,19 +178,19 @@ def timestamps(self, timestamp_list: List[TimeseriesDataTimestamp]) -> None:
             Sets the data as rows of TimeseriesDataTimestamp
         """
 
-        raise NotImplemented("TODO")
+        raise NotImplemented("To be implemented in upcoming versions")
         dotnet_list = System.Collections.Generic.List[Quix.Streams.Streaming.models.timeseriesdataTimestamp]([])
         for item in timestamp_list:
             dotnet_item = item.convert_to_net()
             dotnet_list.Add(dotnet_item)
 
         self.__wrapped.Timestamps = dotnet_list
 
-    def to_panda_dataframe(self) -> pd.DataFrame:
+    def to_dataframe(self) -> pd.DataFrame:
         """
-        Converts TimeseriesData to Panda DataFrame
+        Converts TimeseriesData to pandas DataFrame
 
-        :return: Converted Panda DataFrame
+        :return: Converted pandas DataFrame
```

Comment:
this is not consistent with following lines (like in 268) where pandasDataFrame is used. Which one do we go for?